utils/polus-imagej-util/{{cookiecutter.project_slug}}/src/requirements.txt

```diff
@@ -0,0 +1,2 @@
+bfio[all]==2.1.9
+filepattern==1.4.7
```

Comment:
The purpose of the requirements file is to include all the necessary tools to run the code, even outside a plugin container. I added in `bfio` and `filepattern`, but we should also add in `jpype` and the current version of `pyimagej` we are using.

Reply:
I have created a requirements.txt file with `jpype1`, `pyimagej`, `bfio` and `filepattern`